src/Level.h

```diff
@@ -11,26 +11,41 @@
 
 #include <filesystem>
 #include <string>
+#include <utility>
 
 namespace openblack
 {
+
 class Level
 {
 public:
-	Level(std::string name, std::filesystem::path path, bool isCampaign)
+	enum class LandType : uint32_t
+	{
+		Skirmish = 0,
+		Campaign = 1,
+	};
+
+	Level(std::string name, std::filesystem::path path, std::string description, LandType landType, bool isValid = false)
 	    : _name(std::move(name))
 	    , _scriptPath(std::move(path))
-	    , _isCampaign(isCampaign)
-	{
-	}
+	    , _description(std::move(description))
+	    , _landType(landType)
+	    , _isValid(isValid) {};
```

Comment:
The definition can also move to the cpp file.
This way we can remove the `<utility>` include from a public header.